README.md

```diff
@@ -69,3 +69,6 @@ Combination of `cloudfront:setup` and `cloudfront:build:deploy` commands with or
 ### `serverless:remove`
 
 Remove an entire stack configured in `serverless.yml` via CloudFormation.
+
+CloudFront URL: http://d3id07bk6bnblj.cloudfront.net
+S3 Bucket website endpoint: http://cloud-react-bucket-automated.s3-website-eu-west-1.amazonaws.com/
```

Comment:
Opening this URL should show a 403 Access Denied error according to the task, but I can open it just fine. Please change the bucket policy so that the URL cannot be accessed by principals other than the Cloudfront distribution.

Reply:
I've changed the bucket policy, so now it shows a 403 Access Denied for S3 Bucket website endpoint.